docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
 
-A validator can additionally collect a commission fee from its delegators.
+:::info ䷆ Byzantine Generals Problem 
 
-At start, a validator defines a commission rate, a max change rate, and a max commission rate.
+The **Byzantine Generals Problem** is a scenario illustrating the difficulty in achieving consensus amongst decentralized parties. 
 
-Then it can change its commission rate once every day by at most its max change rate as long as the commision rate doesn’t become greater than the max commission rate.
+The scenario describes several generals surrounding the city of Byzantium and readying for an attack.
 
-Validators are ranked based on the amount of tokens that have been delegated to them, including their self-delegations.
+In order to succeed, all the generals must attack at the same time. Therefore, they must agree a time to attack collectively. However, communication channels are insecure and messages may be intercepted by traitorous generals. Of **n** generals there are **m** traitors. The challenge is to find an algorithm that ensures a successfuly coordinated attack. 
 
-At each block, the top 100 validators who are not jailed are said to be bonded.
+Various possibilities are considered and ruled out. In short, Lamport, Shostak, and Pease found that success was only assured when honest generals outnumbered traitors (**m**) by a number greater than **3m + 1** traitors. In other words, coordination or consensus is impossible to achieve if a third or more of the generals are traitors.
 
-If, for any reason, a validator goes outside the bonded validators set, it enters an unbonding period.
-A validator that has failed to get itself back into the bonded validators set within 21 days is said to be unbonded.
-The number 100 can be updated via a simple governance proposal. – Need testing and verification.
+:::
 
-**Only bonded validators can propose or sign blocks.
-Only bonded validators can earn block rewards.
-Only bonded validators’ tokens are subject to slashing.**
+### Incentivisation
 
-Although the actual act of slashing might occur during the unbonding period.
+Second, incentivisation is built in to the tokenomics model to encourage good behaviour. **Validator operators** who secure the network by running a **Validator node** are required to **stake or self-delegate** a minimum of **2M CUDOS** in return for network rewards. Staking or self-delegating higher amounts increases the chances of that Validator being selected to perform block validation as well as increasing their rewards (as a proportion of the overall amount staked on the network). 
 
-## Redelegation 
+Validators and Delegators earn block rewards for keeping the blockchain operational and secure.
+**Block rewards** consist of **Staking** rewards and **Transaction fees**.
```

Comment:
This may be just nitpicking, but I would make the whole phrase of "Staking rewards" in bold, just like we're not separating "Transaction" and "fees".

Reply:
Done.